socorro/processor/cache_manager.py

```diff
@@ -91,15 +93,19 @@ def __init__(self):
         self._generator = None
         self.inotify = None
         self.watch_flags = (
-            flags.ACCESS
```

Comment:
I should have used OPEN instead of ACCESS originally. When a file is used, there's (usually) one OPEN and then a bazillion ACCESS events, so it floods the inotify queue which is why we got a few IN_Q_OVERFLOW events earlier. This should fix most/all the issues we're having now.